peps/pep-0748.rst

```diff
@@ -0,0 +1,922 @@
+PEP: 748
+Title: A Unified TLS API for Python
+Author: Joop van de Pol <joop.vandepol@trailofbits.com>,
+        William Woodruff <william@yossarian.net>
+Sponsor: Alyssa Coghlan <ncoghlan@gmail.com>
+Discussions-To: https://discuss.python.org/t/pre-pep-discussion-revival-of-pep-543/51263
```

Comment:
When this ready, please open a new discussion and list it here.

You can either open the discussion just before merge and add it here, or remove this line for merge, then open the discussion, and open a new PR to list it here (and in the Post-History).